code/modules/urist/datums/uristvirus2.dm

```diff
@@ -0,0 +1,325 @@
+	E1.stage = 1
+	effects += E1
+	var/datum/disease2/effect/nausea/E2 = new()	// Nausea Stomach, Stage 2.
+	E2.stage = 2
+	effects += E2
+	var/datum/disease2/effect/vomiting/E3 = new()	// Vomiting, Stage 3.
+	E3.stage = 3
+	effects += E3
+	var//datum/disease/effect/stomachparalysis/E4 = new()	// Stomach Big Pain Stage 4.
+	E4.stage = 4
+	effects += E4
+
+// Salmonella: Caused by eating raw meat, certain rotten food, etc. Causes upset stomach, vomitting, annoying messages.
+/datum/disease2/disease/salmonella
+	infectionchance = 0
+	speed = 3
+	spreadtype = "Contact"
+	var/max_stage = 4
+	var/list/affected_species = list(SPECIES_HUMAN,SPECIES_TESHARI,SPECIES_SKREL) // It'd be funny if Vox got Salmonella.
+
+/datum/disease2/disease/salmonella/New()
+	..()
+	var/datum/disease2/effect/stomach/E1 = new()	// Minor Upset Stomach, Stage 1.
+	E1.stage = 1
+	effects += E1
+	var/datum/disease/effect/dehydration/E2 = new()	// Dehydration, Stage 2.
+	E2.stage = 2
+	effects += E2
+	var/datum/disease/effect/choking/E3 = new()	// Nausea, Stage 3.
+	E3.stage = 3
+	effects += E3
+	var/datum/disease2/effect/vomiting/E4 = new()	// Vomiting Stage 4.
+	E4.stage = 4
+	effects += E4
+
+// Mycrosohyosi: (I made this up) Rarely caused by eating rotten food, Causes the person to cough rapidly, shortness of breath and oxygen deprivation.
+/datum/disease2/disease/sporehurl
+	infectionchance = 10 // Spores get coughed out in proximity.
+	speed = 2
+	spreadtype = "Proximity"
+	var/max_stage = 3
+	var/list/affected_species = list(SPECIES_HUMAN,SPECIES_SKRELL,SPECIES_DIONA) // Dionae are plant guys.
+
+/datum/disease2/disease/sporehurl/New()
+	..()
+	var/datum/disease/effect/shortbreath/E1 = new()	// Shortness of Breath - Stage 1.
+	E1.stage = 1
+	effects += E1
+	var/datum/disease/effect/moldtaste/E2 = new()	// Tasting Mushrooms Stage 2.
+	E2.stage = 2
+	effects += E2
+	var/datum/disease/effect/choking/E3 = new()	// Choking Stage 3.
+	E3.stage = 3
+	effects += E3
+	var/datum/disease/effect/sporehurl/E4 = new()	// Infectious Spores Stage 4.
+	E4.stage = 4
+	effects += E4
+
+// Gut Burrowers: Rarely caused by eating rotten food, will slowly cause damage to your stomach, and inflict constant pain messages until cured.
+/datum/disease2/disease/gutworm
+	infectionchance = 0 // To avoid half the station smashing into Surgery
+	speed = 3
+	spreadtype = "Proximity"
+	var/max_stage = 4
+	var/list/affected_species = list(SPECIES_HUMAN,SPECIES_TESHARI,SPECIES_SKRELL,SPECIES_UNATHI)
+
+/datum/disease2/disease/gutworm/New()
+	..()
+	var/datum/disease2/effect/parasitesgrowth/E1 = new()	// Parasite Growing.
+	E1.stage = 1
+	effects += E1
+	var/datum/disease/effect/parasiteorgan/E2 = new()	// Parasite Small Pain
+	E2.stage = 2
+	effects += E2
+	var/datum/disease/effect/parasitesevere/E3 = new()	// Parasite Severe Pain
+	E3.stage = 3
+	effects += E3
+	var/datum/disease2/effect/organs/E4 = new()	// Infectious Spores Stage 4.	TODO This has serious balance considerations, as this is LETHAL.
```

Comment:
The part before TODO seems like an overlooked copy/paste artifact.

Reply:
Nah, I've been considering whether to keep the organ one, since it's very iffy balance wise.